packages/material-ui/src/locale/index.js

```diff
@@ -1138,14 +1230,18 @@ export const ptPT = {
 };
 
 export const roRO = {
+  localization: {
+    formatNumber: formatNumber('ro-RO'),
+  },
   props: {
     // MuiBreadcrumbs: {
     //   expandText: 'Show path',
```

Comment:
@raduchiriac Thank you for the initial contribution. We have a couple of new keys, if you have some time, it would be awesome to translate them :).